package.json

```diff
@@ -1072,6 +1072,7 @@
     "deepmerge": "^4.2.2",
     "del": "^6.1.0",
     "diff": "^5.1.0",
+    "dotenv": "^16.4.5",
```

Comment:
@azasypkin first time doing that. Is there any guide or steps about how to proceed? Thanks! :)

Reply:
Sure! There's nothing particularly complex here, and here are the steps:

1. If you're introducing a new dependency, decide which team is supposed to own it (see point `6` in our [dev docs](https://docs.elastic.dev/kibana-dev-docs/standards#adding-dependencies) to understand what ownership means). Usually, it's the team you belong to.
2. Check if the dependency in question is already owned by another team in [`renovate.json`](https://github.com/elastic/kibana/blob/main/renovate.json). It might be the case that we had this dependency before, removed it, but didn’t update the ownership. If it's already present in `renovate.json`, it's safe to clean that up.
3. If you don’t yet have a `group` for the dependencies owned by your team, add a new one [like this](https://github.com/elastic/kibana/blob/1a92a4ba136dc0e3488c06b25d1c96ec01c638cc/renovate.json#L215-L234):

```json
    {
      "groupName": "platform security modules", <--- Arbitrary name, just for your convenience 
      "matchDepNames": [ <--- The list of the dependencies to own
        "css.escape",
        "node-forge",
        "formik",
        "@types/node-forge",
        "require-in-the-middle",
        "tough-cookie",
        "@types/tough-cookie",
        "xml-crypto",
        "@types/xml-crypto",
        "@kayahr/text-encoding"
      ],
      "reviewers": ["team:kibana-security"], <--- Which GitHub team should be assigned to review the Renovate PR once it’s automatically created to bump the dependency version
      "matchBaseBranches": ["main"], <--- What branches to track, we only care about `main` at this point
      "labels": ["Team:Security", "release_note:skip", "backport:all-open"], <--- What GitHub labels to add to the Renovate PR, I suggest keeping both "release_note:skip" and "backport:all-open"
      "minimumReleaseAge": "7 days", <--- I suggest sticking to this number too
      "enabled": true <--- Make sure your group is `enabled` :)
    },
```

Let me know if you have more questions!

Reply:
@azasypkin done! lots of thanks for sharing the steps :)